src/SprykerSdk/Acp/Manifest/Reader/AppManifestReader.php

```diff
@@ -114,7 +118,7 @@ protected function extendCollectionByConfiguration(
             ->setConfiguration(json_decode((string)file_get_contents($configurationFile->getPathname()), true));
 
         if (json_last_error()) {
-            return;
+            throw new InvalidConfigurationException(json_last_error_msg());
```

Comment:
Please make the messages more verbose and helpful. I expect something like: `There was an error while decoding the configuration file "/path/to/file". Error: ...`

With this, the user would directly see which file is in question and what exactly happened.